pkg/lib/cockpit-components-password.jsx

```diff
@@ -99,7 +99,7 @@ export const PasswordFormFields = ({
                            <Popover bodyContent={password_label_info}>
                                <button onClick={e => e.preventDefault()}
                                        className="pf-v5-c-form__group-label-help">
-                                   <HelpIcon noVerticalAlign />
+                                   <HelpIcon />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19537-20231027-075124-5a132abc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)